tests/data/devices/tze200-a7sghmms-ts0601.json

```diff
@@ -0,0 +1,897 @@
+{
+  "version": 1,
+  "ieee": "ab:cd:ef:12:b5:0f:0b:2f",
+  "nwk": "0x5C8B",
+  "manufacturer": "_TZE200_a7sghmms",
```

Comment:
Hmm, interesting that it affected this Tuya valve with a v2 quirk. We have a local `Metering` cluster for that and it's one of the few times we actually do not use `kW(h)` as the unit for it, but instead water/liters:
https://github.com/zigpy/zha-device-handlers/blob/928268cca5d22a048e72d45ae1c24a1b6dd92a9c/zhaquirks/tuya/builder/__init__.py#L128-L148